syntax = "proto3";

message Inventory{
    string Inv = 1;
}

message Request{
}

message LastIdd{
    string LIdd = 1;
}


message product {
    string id = 3;
    string name = 4;
  }

message TransactionResponse{
    int32 status_code = 1;
}

service InventoryService {
    rpc GetInventory(Request) returns (Inventory){}
    rpc GetLastIdd(Request) returns (LastIdd){}
    rpc addProducts(product) returns (TransactionResponse) {}
}